bxsolana/provider/http.ts

```diff
@@ -43,6 +45,7 @@ import {
 } from "../proto/messages/api/index.js"
 import { BaseProvider } from "./base.js"
 import config from "../../utils/config.js"
+import { IsRPCError, RPCError } from "./error.js"
```

Comment:
Are you missing a file here?

BTW, naming: function should be `isRpcError`, and class should be `RpcError`. TS doesn't capitalize acronyms

Reply:
Yes! I didn't realize it 😅